supervisor/v1/supervisor.proto

```diff
@@ -70,6 +70,12 @@ message DeploymentConfig {
     // The replicas associated with this deployment. This field is only
     // valid if the deployment is a master deployment.
     repeated ReplicaAndLocation replicas = 5;
+
+    // The location where to place the data, determined by the CP with the appropriate storage pool
```

Comment:
Please can we add a bit more detail about how `blob_paths` is used and what happens under different scenarios. This might be better placed in the `ConfigureDeployment` documentation? For example:
* explain that on a call to ConfigureDeployment the dataplane will use the paths specified in `blob_paths` to house the blob files,
* explain that on a subsequent call to ConfigureDeployment if there are any new dirs in `blob_paths` the dataplane will create new blob files under those directories and start writing new data to them
* explain what happens if a path is removed between subsequent calls to `blob_paths` (I can't even remember what we do here?)
* anything else where we've made a non-obvious design decision that is worth documenting.